src/vocabularyChecker/detectWrongSyntaxOld.ts

```diff
@@ -0,0 +1,144 @@
+import json from "./AllowedVocab.json";
```

Comment:
What is this file used for? With `git` you don't have to keep code that is no longer used - if you ever need to go back to it, you can look at the history.